### PR TITLE
Add destroy to services

### PR DESCRIPTION
**Pull request by beingSCK**

### CODE REVIEW

File: app/views/services/show.html.erb [hunk 1]
```diff
@@ -24,12 +24,18 @@
...
           <% end %>
 
           <!-- no link for booking new yet so i left a second index path -->
           <%= link_to new_service_booking_path(@service) do%>
-          <p class="btn btn-primary">Book Service</p>
+          <p class="btn btn-success">Book Service</p>
+          <% end %>
+
+          <% if @service.user.id == @user.id %>
+            <%= link_to service_path(@service.id), method: :delete do%>
```
Comment (Grokepau) [inline app/views/services/show.html.erb]: can you add a confirmation :
method: :delete,
        data: { confirm: "Are you sure?" }

**Comment by Grokepau** [comment]
and did you set a route?

File: app/views/services/show.html.erb (hunk 1 above)
Comment (beingSCK) [reply app/views/services/show.html.erb]: I literally was going to look up how to add a confirm pop-up when I got in today! 

I'l do it now and re-push.

Comment (beingSCK) [reply app/views/services/show.html.erb]: done! See next commit.

**Pull request merged**